src/s3-copy.py

```diff
@@ -26,17 +26,63 @@
 cycling_interval = config_dict.get('cycling_interval')
 # read source info
 sources = config_dict.get('source')
+# read destination info
+destination = config_dict.get('destination')
+
+#----- create source clients
+print('Opening aws resources')
+destination_bucket = boto3.Session(profile_name=destination.get('credential_profile')).resource('s3').Bucket(destination["bucket"])
+destination_fn_template = destination.get('path')+destination["file_template"]["prefix"]+destination["file_template"]["root"]+destination["file_template"]["suffix"]
+print('...Done')
+
+#----- open log files
```

Comment:
There exists an entire class to handle logging. Why are you not using it?

An example can be seen here: 

https://github.com/noaa-psd/UFS-RNR/blob/feature/ush_reorg/ush/produtil/logger_interface.py

Reply:
Good point. I looked up the logging class for writing out which files are copied but chose to write to file explicitly. Yes. for trace messages i should use the logger class.